src/AutorestSwift/Models/Complex/Request.swift

```diff
@@ -75,9 +87,7 @@ extension Request {
                 bodyParams.append(param)
             }
         }
-        // current logic only supports a single request per operation
-        assert(bodyParams.count <= 1, "Unexpectedly found more than 1 body parameters in request... \(name)")
```

Comment:
This assertion is still relevant and shouldn't be removed.

Reply:
The assert is not removed. It is just moved. See line 72 of the src/AutorestSwift/Models/Complex/Request.swift

```
	        // current logic only supports a single request per operation
        assert(bodyParams.count <= 1, "Unexpectedly found more than 1 body parameters in request... \(name)")
        return bodyParams.first
    }
```